docker/docker-compose.yml

```diff
@@ -0,0 +1,54 @@
+services:
+  database:
+    image: postgres
```

Comment:
carefully choose Postgres version to ensure compatable with current Hibernate, flyway version